ts/nni_manager/training_service/reusable/aml/amlClient.ts

```diff
@@ -135,11 +135,19 @@ export class AMLClient {
     
     // Monitor error information in aml python shell client
     private monitorError(pythonShellClient: PythonShell, deferred: Deferred<any>): void {
+        pythonShellClient.on('stderr', function (error: any) {
```

Comment:
The argument is not "error". It's stderr's content.
https://github.com/extrabacon/python-shell/blob/master/index.ts#L442

Reply:
I plan to split this PR into smaller ones. Let's discuss later.